_extensions/mpim/slide_background.css

```diff
@@ -0,0 +1,17 @@
+.quarto-title-block .slide-background-content {
+    background-image: url("titles/green.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.level1 .slide-background-content {
+    background-image: url("slide_separators/dots.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+}
```

Comment:
I fear that this breaks the mobile representation of the title page. It seems that background images do not stay within the viewport
<img width="636" alt="Screenshot 2024-06-26 at 10 07 36" src="https://github.com/mpimet/quarto/assets/9482218/a9d808d8-2b98-4fb8-8547-1796fb442917">

Reply:
Argh 😬 good catch, I don't know why I didn't notice this. Should now be fixed.